deps.edn

```diff
@@ -1,14 +1,23 @@
 {:paths   ["src"]
- :deps    {org.clojure/clojure       {:mvn/version "1.10.3"}
-           org.clojure/clojurescript {:mvn/version "1.11.4"}
+ :deps    {org.clojure/clojure       {:mvn/version "1.11.1"}
+           org.clojure/clojurescript {:mvn/version "1.11.60"}
            org.clojure/core.async    {:mvn/version "1.5.648"}}
  :aliases {:cljs   {:extra-deps  {thheller/shadow-cljs {:mvn/version "2.17.2"}
                                   binaryage/devtools   {:mvn/version "1.0.4"}}
                     :extra-paths ["test"]}
            :test   {:extra-deps  {lambdaisland/kaocha                  {:mvn/version "1.63.998"}
-                                  io.github.cognitect-labs/test-runner {:git/tag "v0.5.0"
-                                                                        :git/sha "48c3c67"}}
+                                  io.github.cognitect-labs/test-runner {:git/tag "v0.5.1"
+                                                                        :git/sha "dfb30dd"}}
                     :extra-paths ["test"]}
+           :jvm-test {:extra-deps  {io.github.cognitect-labs/test-runner {:git/tag "v0.5.1"
+                                                                          :git/sha "dfb30dd"}}
+                      :extra-paths ["test"]
+                      :main-opts ["-m" "cognitect.test-runner"]
+                      :exec-args {:nses [superv.async-test]
+                                  :dirs ["test"]}
+                      :exec-fn cognitect.test-runner.api/test}
+           :cljs-test {:extra-paths ["test"]
+                       :main-opts ["-m" "cljs.main" "-t" "node" "-m" "superv.node-runner"]}
```

Comment:
Did you see the `bin/run-cljstests` script? I know it is not running the node-tests atm. But when you uncomment https://github.com/replikativ/superv.async/blob/main/test/superv/async_test.cljc#L332 you should be able to run the node-tests from the script again and this part here would not be necessary.

Reply:
Then our pipeline for superv.async looks better again. That would be ace!